web/impact/impact/permissions/v1_api_permissions.py

```diff
@@ -1,12 +1,23 @@
 from accelerator_abstract.models.base_user_utils import is_employee
 from impact.permissions import (
     settings,
-    BasePermission)
+    BasePermission,
+)
+
+SAFE_METHODS = ['GET', 'HEAD', 'OPTIONS']
+
+
+def is_object_owner(request, user_id):
+    if user_id and request.method in SAFE_METHODS:
+        return str(user_id) == str(request.user.id)
 
 
 class V1APIPermissions(BasePermission):
     authenticated_users_only = True
 
     def has_permission(self, request, view):
+        user_id = request.parser_context['kwargs'].get('pk', '')
```

Comment:
why is this called "user_id"? It looks like it's the object of any object we're querying on. Is this only ever called with a user as the object?

Reply:
I should probably make sure that the user is on the user detail page before getting that.